cli/src/main/java/com/liferay/blade/cli/BladeCLI.java

```diff
@@ -761,6 +874,70 @@ private void _migrateBladeSettingsFile(File settingsFile) throws IOException {
 		}
 	}
 
+	private void _promptForMissingMainParameter(Object commandArgs, List<String> argsCollection) {
```

Comment:
This method should just return the value of the missing parameter.  It should not mutate a passed in collection.